.github/workflows/release.yml

```
@@ -40,26 +40,21 @@ jobs:
         run: |
           echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
           npm whoami
-      - name: Extract version from branch name
-        run: echo "::set-output name=version::${GITHUB_REF##*/}"
-        id: extract_version
+      - name: Extract branch from git ref
+        run: |
+          echo "::set-output name=name::${GITHUB_REF#refs/*/}"
+          echo "::set-output name=version::${GITHUB_REF##*/}"
+        id: extract_branch
       - name: Releaseing with lerna
         # also see some options in lerna.json
-        run: npx lerna publish ${{ steps.extract_version.outputs.version }} --conventional-commits --create-release github --yes --no-private
+        run: npx lerna publish ${{ steps.extract_branch.outputs.version }} --conventional-commits --create-release github --yes --no-private
         env:
           GH_TOKEN: ${{ github.token }}
       - name: Create Pull Request
-        id: cpr
-        uses: peter-evans/create-pull-request@v3
-        with:
-          base: ${{ github.event.push.repository.default_branch }}
-          branch: release/chores-into-default-branch
-          delete-branch: true
-          title: 'chore(release): publish'
-          body: 'This is an automated pull request to merge some chores into the default branch.'
-          labels: |
-            automated pr
-      - name: Check Pull Request
         run: |
```

```suggestion
        run: >
```

も入れないとかもです！（上のコメントで複数行指定した時に一緒に入れられませんでした😂）

---

ほんとだ！！ｗ